### DISCUSSION TITLE
Can i use pelorus with Kubernetes other than OpenShift ?

### DISCUSSION THREAD

**Discussion by geowalrus4gh** [dora-metrics/pelorus#1126] [General]
Can I install and use pelorus with other Kubernetes providers like AWS EKS ?

**Comment by mpryc** [comment]
Currently Pelorus is available for OpenShift only. It's because it uses operators from marketplace and some specific to OpenShift APIs with the Helm chart templating.

Core exporters, prometheus could be made available for k8s, but it never was done. Contributions are welcome if that's required.
